Neos.Media/Classes/Domain/Service/AssetService.php

```diff
@@ -88,6 +93,30 @@ class AssetService
      */
     protected $imageService;
 
+    /**
+     * @Flow\InjectConfiguration(path="variantPresets", package="Neos.Media")
+     * @var array
+     */
+    protected $variantPresetsConfiguration = [];
+
+    /**
+     * @var VariantPreset[]
+     */
+    private $variantPresets = [];
+
+    /**
+     * @return VariantPreset[]
+     */
+    public function getVariantPresets(): array
```

Comment:
Wouldn't this be more related to the `AssetVariantGenerator` than this? It feels a bit lost and unrelated here?

Reply:
Indeed.